appeals/web/src/server/appeals/question-page/question-page.controller.js

```diff
@@ -0,0 +1,74 @@
+import { getAppealDetailsFromId } from '#appeals/appeal-details/appeal-details.service.js';
+import { getLpaQuestionnaireFromId } from '#appeals/appeal-details/lpa-questionaire/lpa-questionnaire.service.js';
+import { appealShortReference } from '#lib/appeals-formatter.js';
+import logger from '#lib/logger.js';
+import {
+	appealBackLink,
+	appealQuestionPage,
+	lpaQBackLink,
+	lpaQQuestionPage
+} from './question-page.mapper.js';
+
+/**
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ */
+const renderQuestionPage = async (request, response) => {
+	try {
+		const currentUrl = request.originalUrl;
+		const origin = currentUrl.split('/')[currentUrl.split('/').length - 2];
```

Comment:
Very minor, but you could save the result of the split into a variable here so you don't need to do the split twice